app/celery/scheduled_tasks.py

```diff
@@ -32,13 +32,11 @@
 from app.dao.notifications_dao import (
     dao_old_letters_with_created_status,
     dao_precompiled_letters_still_pending_virus_check,
-    is_delivery_slow_for_providers,
```

Comment:
Looks like this isn't used anywhere else, so we should be able to delete it and its tests, as well.